source/agora/utils/PrettyPrinter.d

```diff
@@ -453,13 +452,15 @@ Outputs (8):
 GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000),
 GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000),
 GCOQ...LRIJ(61,000,000), GCOQ...LRIJ(61,000,000)
-====================
+====================================================
```

Comment:
It's used for separating each item of a range like in the case of transactions In a block. So it should remain a shorter string as it was.

Reply:
As we discussed is ok.